libpympb/pympb.hpp

```diff
@@ -9,6 +9,16 @@
 
 namespace py_mpb {
 
+// A global "verbosity" level.
+//    0: minimal output
+//    1: a little
+//    2: a lot (default)
+//    3: debugging
+//
+// This is redeclared here so SWIG will see it.
+extern "C" int mpb_verbosity;
```

Comment:
Now that https://github.com/NanoComp/mpb/commit/3108059ef78b57ba466fb3dce240c26c6844045e is merged, you can protect this with
```c
#if MPB_VERSION_MAJOR > 1 || (MPB_VERSION_MAJOR == 1 && MPB_VERSION_MINOR >= 11)
```

Reply:
Since SWIG currently doesn't see the values of those macros I made a corresponding change in `pympb.cpp` instead. Although it's possible to make those available to SWIG, this seemed to be the simplest change to make things work with old and new libmpb.